test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -580,10 +579,15 @@ public enum ShowWindowCommands : int
         $ps | Stop-Process -Force
     }
 
-    It "Invalid -WindowStyle returns error" {
+    It "Invalid -WindowStyle returns error on Windows" -Skip:(!$IsWindows) {
         pwsh -WindowStyle invalid
         $LASTEXITCODE | Should Be $ExitCodeBadCommandLineParameter
     }
+
+    It "-WindowStyle returns error on Linux and macOS" -Skip:$IsWindows {
+        pwsh -WindowStyle
+        $LASTEXITCODE | Should Be $ExitCodeBadCommandLineParameter
```

Comment:
This test would pass even without your change
```
S:9> pwsh -WindowsStyle                                                                            
The argument '-WindowsStyle' is not recognized as the name of a script file. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
PS:10> $LASTEXITCODE                                                                                
64
```
Can you please update the test to make it pass only after your change?

Reply:
Are you okay with a string match here? Looks like only one exit is available and you don't have much to go on invoking a native command. Adding an additional exit code seems like overkill but I can do it if that is the direction we want to go.

Reply:
@JamesWTruher @adityapatwardhan Can you please suggest what might be a good test in this case?

Reply:
After a second thought, I don't feel it adds much value to add a test for this change. It requires spending more time on a simple change that's unlikely to regress.
I'm fine that you just revert the changes in `ConsoleHost.Tests.ps1`. I have manually verified the changes on Linux, and the error message looks good:
```
bash:1> ./pwsh -WindowStyle invalid
Parameter -WindowStyle is not implemented on this platform.
```

Reply:
@daxian-dbw ConsoleHost.Tests.ps1 reverted.

Reply:
@thezim Thank you!